model/Licensing/Licensing.md

```diff
@@ -0,0 +1,97 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Licensing
+
+## Summary
+
+The Licensing Profile defines a minimum set of license information required to comply with typical license compliance use cases.
+
+## Description
+
+The Licensing profile only contains additional restrictions.
+Classes and Property restrictions are defined in the SimpleLicensingProfile
+(Classes and Properties associated with string license expressions) and in the ExpandedLicensingProfile (Classes and Properties used for a
+fully parsed syntax tree of license expressions).
+
+There are 2 relationships which are required for any Software Artifact - declaredLicense and concludedLicense.
```

Comment:
This isn't what we had decided would be required for the Licensing profile as a requirement during the 2020/2021 discussions. Only concludedLicense was required.

That said, I'm not clear what it means for it to be "required" in any case where concludedLicense is a relationship now, and where I don't think documents are required?

I'm not sure how other profiles are handling requirements like this, where relationships might exist anywhere in SPDX data not tied to a particular document. Are other profiles scoping it to e.g. "in order to conform with a profile, there must be an SPDX Document which contains a relationship for each of these items"?

Reply:
And sorry, just to clarify -- only concludedLicense was required _for the SPDX data to conform to the Licensing profile_.

Reply:
On your first point - I fixed it to only state the requirement was on the concluded license.

On you second point, I opened issue #463 to track.  This is targeted to be resolved after RC2 whereas these descriptions should be updated prior to release of RC2.